### PR TITLE
fixed problem with numeric cells in PythonFileReader

### PR DESCRIPTION
**Pull request by arodiss**

### CODE REVIEW

File: Xls/Reader/excel_openpyxl.py [hunk 1]
```diff
@@ -25,7 +25,11 @@ def run(argv):
     for row in sheet.iter_rows(row_offset=int(args.start) - 1):
         current_row_read = []
         for cell in row:
-            current_row_read.append(cell.value.encode('utf-8'))
+            value = cell.value
+            if isinstance(value, (int, long)):
```
Comment (tier2003) [inline Xls/Reader/excel_openpyxl.py]: why not just 
```
value = str(cell.value).encode('utf-8')
``` 
?

Comment (arodiss) [inline Xls/Reader/excel_openpyxl.py]: @tier2003 
That would be nice, however in `Python2`:
```
>>> str(u'\ua000').encode('utf-8')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeEncodeError: 'ascii' codec can't encode character u'\ua000' in position 0: ordinal not in range(128)
```

Comment (tier2003) [inline Xls/Reader/excel_openpyxl.py]: Are you have using Python2 anywhere?

Comment (arodiss) [inline Xls/Reader/excel_openpyxl.py]: @tier2003 
It's not "me", it's open-source lib

Comment (tier2003) [inline Xls/Reader/excel_openpyxl.py]: How did you reproduce it? )
Lib is open-sourced - ok. But you`re running on your/our servers which have the latest python versions )
If the last is true, then the code above looks like overcheck.

Comment (arodiss) [inline Xls/Reader/excel_openpyxl.py]: @tier2003 
If you want to go in such details, then listen.
I call `python` executable which is convenient and cross-platform. But for our system`python` is alias for `python2.7`. New version is indeed available, but only through the non-convenient name `python3`
In this lib you can pass custom executable name, but I don't want to spend my time on using non-default cofiguration

Comment (tier2003) [inline Xls/Reader/excel_openpyxl.py]: of course, it`s up to you, but from my POV, it is better to make custom config once than adding all these annoying checks and fixes )

Comment (arodiss) [inline Xls/Reader/excel_openpyxl.py]: @tier2003 
Oh yes, fixes are annoying :smile:

**Pull request merged**